SpreadCheetah/SpreadCheetah.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net6.0;net7.0;net8.0</TargetFrameworks>
```

Comment:
We should not drop support for .NET Standard 2.0. I assume you removed it because of `DateOnly`, which is only available on .NET 6 and later. Using `DateOnly` would be nice for this feature, but I don't think it is enough to drop the backwards compatibility. So for now we should basically only use `DateTime` for this feature.

Reply:
No problems. I will revert compatibility in the next commit.